pennylane/devices/default_mixed.py

```diff
@@ -95,7 +130,9 @@ def analytic_probability(self, wires=None):
         rho = self._reshape(self._state, (2 ** self.num_wires, 2 ** self.num_wires))
         # probs are diagonal elements
         probs = self.marginal_prob(self._diag(rho), wires)
-        return probs
+
+        # take the abs so that probabilities are not shown as complex numbers
+        return self._abs(probs)
```

Comment:
Just curious: they would have zero imaginary parts, correct?

Reply:
Would it be better/less overhead to just do `self._real`?